### PR TITLE
Added ordersModel which directs to correct create function for a given product

### PR DESCRIPTION
**Pull request by bartojer**

### CODE REVIEW

**Review by NicholasBoss — APPROVED** [review]

**Review by NicholasBoss — CHANGES_REQUESTED** [review]
>

File: controllers/ordersController.js [hunk 1]
```diff
@@ -32,4 +32,21 @@ ordersController.buildDelete = async function(req, res){
...
+            product_name,
+            measurement_name,
+            size_type,
+            fastener_type,
+            color_name,
+            mesh_type,
+            mirage_build_out,
+            mirage_3500_handle
+        )
+    }
```
Comment (NicholasBoss) [inline controllers/ordersController.js:49]: You need an except block here.

**Review by NicholasBoss — CHANGES_REQUESTED** [review]
Please check the comments in the ordersController file.

File: controllers/ordersController.js (hunk 1 above)
Comment (bartojer) [reply controllers/ordersController.js:49]: Yes, you're right. I should've commented it out or not synced it until it was complete/not throwing an error. I'm glad you noticed.

**Review by NicholasBoss — APPROVED** [review]

**Pull request merged**